Source/capture.cpp

```diff
@@ -57,40 +58,31 @@ FILE *CaptureFile(std::string *dstPath)
 	return OpenFile(dstPath->c_str(), "wb");
 }
 
-/**
- * @brief Make a red version of the given palette and apply it to the screen.
- */
-void RedPalette()
+} // namespace
+
+std::string FileNameForChat(const std::string &fullPath)

```

Comment:
Why isn't this in the anonymous namespace?

Reply:
Idk I just work here